docs/changelog.md

```diff
@@ -6,6 +6,21 @@ sidebar_label: API Changelog
 
 ⚠ Consider subscribing to our [**API Newsletter**](http://eepurl.com/g2EiC1) to be notified about upcoming API changes in the future.
 
+## Version 1.12
+The following changes to the API are scheduled to be deployed in **calendar week 49**.
```

Comment:
Please keep the spacing consistent, for example, add an empty line after headers (`##` and `###`)